.gitignore

```diff
@@ -124,3 +124,7 @@ dist
 # Swap files for vim
 *.swp
 *.swo
+
+
+# Generated code
+crates/xmtp_proto/src/gen
```

Comment:
Whoops. I changed this folder from `xmtp_proto` to `xmtp-proto` but didn't update the .gitignore